### ISSUE TITLE
Rule request: redundant_optional_initialization

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#1052]
Initializing an optional variable with nil is redundant.

```swift
// OK
var myVar: Int?
let myVar: Int? = nil

// should trigger
var myVar: Int? ↓= nil
```
